xskillscore/core/deterministic.py

```diff
@@ -137,7 +137,7 @@ def pearson_r(a, b, dim=None, weights=None, skipna=False, keep_attrs=False):
         weights,
         input_core_dims=input_core_dims,
         kwargs={"axis": -1, "skipna": skipna},
-        dask="parallelized",
+        dask="allowed",
```

Comment:
I am still in favor of having `dask` as a global keyword, see `xarray.set_config`

Reply:
`~/Coding/xskillscore/asv_bench$ asv continuous -f 1.1 upstream/master HEAD -b deterministic.Compute_small.time_xskillscore_metric_small`

Reply:
```

[ 70.83%] ··· ================================================ ========
                                     m
              ------------------------------------------------ --------
                     <function rmse at 0x7fe9d0353730>          failed
                   <function pearson_r at 0x7fe9d032c400>       failed
                      <function mae at 0x7fe9d0353840>          failed
                      <function mse at 0x7fe9d03537b8>          failed
               <function pearson_r_p_value at 0x7fe9d0353378>   failed
              ================================================ ========
```

Reply:
can do easy checks with `asv dev -b deterministic.Compute_small.time_xskillscore_metric_small`, must delete that encoding it seems